platform/android/MapLibreAndroid/src/cpp/map_renderer.cpp

```diff
@@ -257,7 +257,7 @@ void MapRenderer::onSurfaceCreated(JNIEnv& env, const jni::Object<AndroidSurface
     // If we're running the emulator with the OpenGL backend, we're going to crash eventually,
     // unless we enable this mitigation.
     if (inEmulator()) {
-        renderer->enableAndroidEmulatorGoldfishMitigation(true);
+        renderer->enableAndroidEmulatorGoldfishMitigation(false);
```

Comment:
@mwilsnd For now I just disabled old workaround for Android crash. I would remove the code after you confirm is not crashing on your side too.